data/products/api7/features.json

```
@@ -592,39 +592,589 @@
+              "href": ""
+            }
+          ]
+        },
+        {
+          "title": "Authentication",
+          "list": [
+            {
+              "title": "key-auth",
+              "href": ""
+            },{
+              "title": "basic-auth",
+              "href": ""
+            },{
+              "title": "JWT",
+              "href": ""
+            },{
+              "title": "API Signature Verification (HMAC)",
+              "href": ""
+            },{
+              "title": "OAuth2",
+              "href": ""
+            },{
+              "title": "SSO",
+              "href": ""
+            },{
+              "title": "Auth0, Okta, etc.",
+              "href": ""
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Observability",
+      "list": [
+        {
+          "title": "Metrics",
+          "list": [
+            {
+              "title": "Prometheus",
+              "href": ""
+            }
+          ]
+        },
+        {
+          "title": "Tracing",
+          "list": [
+            {
+              "title": "SkyWalking",
+              "href": ""
+            },
+            {
+              "title": "Zipkin",
+              "href": ""
+            },
+            {
+              "title": "OpenTracing",
+              "href": ""
+            }
+          ]
+        },
+        {
+          "title": "Logs",
+          "list": [
+            {
+              "title": "Kakfa",
+              "href": ""
+            },
+            {
+              "title": "Http Logger",
+              "href": ""
+            },
+            {
+              "title": "TCP Logger",
+              "href": ""
+            },
+            {
+              "title": "UDP Logger",
+              "href": ""
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "Performance and High Availability",
+      "list": [
+        {
+          "title": "Ultimate Performance",
+          "list": [
+            {
+              "title": "Single Core Performance",
+              "href": ""
+            }, {
+              "title": "Minimum latency per request",
+              "href": ""
+            }
+          ]
+        },
+        {
+          "title": "Deployment",
+          "list": [
+            {
+              "title": "Data plane stateless",
+              "href": ""
+            },
+            {
+              "title": "Supports Cluster as Configuration Center",
+              "href": ""
+            }
+          ]
+        },
+        {
+          "title": "Cluster Management",
+          "list": [
+            {
+              "title": "Supports configuration and management of multiple clusters",
+              "href": ""
+            },
+            {
+              "title": "Automatic cluster selection under multi-tier networks",
+              "href": ""
+            }
+          ]
+        },
+        {
+          "title": "Dynamic and hot updates",
```

The others are written with all first letter capitalized, might change this to follow the patter
```suggestion
          "title": "Dynamic and Hot Updates",
```